components/user-mgt/org.wso2.carbon.identity.user.pre.update.profile.action/pom.xml

```diff
@@ -91,13 +121,20 @@
 
                             org.wso2.carbon.context; version="${carbon.kernel.package.import.version.range}",
                             org.wso2.carbon.utils; version="${carbon.kernel.package.import.version.range}",
+                            org.wso2.carbon.user.core.*;version="${carbon.kernel.package.import.version.range}",
+                            org.wso2.carbon.user.api; version="${carbon.user.api.imp.pkg.version.range}",
 
                             org.wso2.carbon.identity.core.context; version="${carbon.identity.package.import.version.range}",
                             org.wso2.carbon.identity.core.context.model; version="${carbon.identity.package.import.version.range}",
                             org.wso2.carbon.identity.core.util; version="${carbon.identity.package.import.version.range}",
                             org.wso2.carbon.identity.action.management.api.exception; version="${carbon.identity.package.import.version.range}",
                             org.wso2.carbon.identity.action.management.api.model; version="${carbon.identity.package.import.version.range}",
-                            org.wso2.carbon.identity.action.management.api.service; version="${carbon.identity.package.import.version.range}"
+                            org.wso2.carbon.identity.action.management.api.service; version="${carbon.identity.package.import.version.range}",
+                            org.wso2.carbon.identity.action.execution.api.*; version="${carbon.identity.package.import.version.range}",
+                            org.wso2.carbon.identity.user.action.api.service; version="${carbon.identity.package.import.version.range}",
+                            org.wso2.carbon.identity.user.action.api.model; version="${carbon.identity.package.import.version.range}",
+                            org.wso2.carbon.identity.claim.metadata.mgt.*; version="${carbon.identity.package.import.version.range}",
+                            org.wso2.carbon.identity.application.authentication.framework.util; version="${carbon.identity.package.import.version.range}",
```

Comment:
Let's sort the imports in alphabetic order. Not required, but better to have. Seems like we missed to review this in the management PR.

Reply:
Is this a practice followed. I rather order having most downstream stuff (core) stuff top

Reply:
Ack.